contrib/kubernetes/README.md

```diff
@@ -0,0 +1,214 @@
+# Kubernetes managing an etcd2 cluster
+
+The goal of this project is to enable Kubernetes to manage an etcd2 cluster and to keep it healthy
+
+It accomplishes this by using persistent storage that will always be available to a given pod even as it is recreated
+
+The example files provided here assume a cluster of size 3, but are generalizable to a any fixed sized cluster
+
+## Architecture
+
+In order for an etcd2 cluster to work without any discovery we need to have static IP addresses that are well known for each server instance.  We accomplish that with creating individual kubernetes' services for each replication controller/pod that will be part of our cluster.  Using kubernetes dns or environment variables lets each pod's etcd server know how to connect to the servers in the cluster.  Each of the N replication controller/pod instances will define a label that specifies app=etcd-N and each of the N services will select on app=etcd-N
```

Comment:
* "We accomplish that by creating..."
* delete apostrophe after "kubernetes'"
* `s/that will be a part of our/in the/`
* DNS
* `s/etcd server/etcd/
* mark the literal strings like `app=etcd-N`
* period at end of last sentence